### ISSUE TITLE
Support serving raw assembly bytes

### ISSUE THREAD

**Issue by mstange** [mstange/profiler-get-symbols#25]
To support an inline assembly code view in the profiler, we should add an "endpoint" / "URL" to request assembly code, or at least the raw bytes so that the front-end can do the disassembly.

JSON format TBD.

`/assemblybytes/v1`

Input data

 - The binary's debugName + breakpadId, or alternatively (or maybe additionally) name + codeId
 - Start address
 - End address? Or should it be the implementation's problem to find the function at the start address and then return the entire function's code?

**Comment by mstange** [comment]
For this API to work, we need to have access to the binary. But we do not necessarily need to have symbols for that binary - or at least I'd like this API to work even when we don't have symbols.

This way, for profiles that were captured on this machine, we can always get assembly code without having to fetch anything from anywhere else; we always have the binary because that's what we were recording.

Being able to get assembly without symbols matters for Windows system libraries, and when profiling official Firefox builds. In both cases, the profiler gets the symbols from the Mozilla symbol server. So the symbols are only on the server. But the assembly is not on the server (or at least not yet) - we want to get the assembly from the locally running binary.

Long story short, what this means is that the API request needs to have a precise start and end address. The profiler UI has obtained symbol information from somewhere (possibly us, possibly some public server), and it knows where the function starts and maybe also has a good guess of where it ends. Then it can just request that address range, and the local symbolication API can look up that range in the text section of the binary without having to understand what it's looking at.

**Comment by mstange** [comment]
I think it would be better to serve the disassembled instructions as strings. I.e. do the disassembly on the server side, not in the front-end. This would place less of a burden on the front-end, and it would make it easier to integrate it with symbolication information (in scenarios where that information is available). For example, call instructions could have the destination function name instead of a raw address.

Example JSON result:

```json
{
  "startAddress": "0x1234abcd",
  "size": "0x345",
  "instructions": [
    [0, "sub sp, sp, #0x20"],
    [4, "stp x29, x30, [sp, #0x10]"],
    [12, "add x29, sp, #0x10"],
    [22, "adr x1, $+0x63028"]
  ]
}
```

**Comment by mstange** [comment]
This was implemented in samply-api and was picked up with the update in 97e9a6ad2ed7024cecf8bde66c73fc5b837c9ec1.
